docs/api_docs/user.md

```diff
@@ -30,60 +35,44 @@
 #### `GET user/logout`
 - Logs user out of Meetify
 - Ends user session
+- Status Codes:
+    - `204` - Successful logout
```

Comment:
Is there a reason this is `204` and not `200` like the others?

Reply:
From my prior understanding, it is pretty common practice to use a 204 for a logout endpoint that simply logs out and doesn't return anything in the response.

Reply:
![image](https://user-images.githubusercontent.com/32147527/108637033-d9803680-7456-11eb-9169-6b4b5af796c7.png)

Ah, looked it up and found this explanation at [this link](https://www.g33kzone.com/2017/05/http-status-codes-200-vs-202-vs-204.html). Didn't know about this convention; good call!